lib/core/src/Cardano/Wallet/Api/Types.hs

```diff
@@ -1026,6 +1034,24 @@ data ApiTransaction (n :: NetworkDiscriminant) = ApiTransaction
     } deriving (Eq, Generic, Show)
       deriving anyclass NFData
 
+-- | The response cardano-wallet returns upon successful submission of a
+-- mint/burn transaction.
+data ApiMintBurnTransaction (n :: NetworkDiscriminant) = ApiMintBurnTransaction
+    { transaction         :: !(ApiTransaction n)
+    -- ^ Information about the mint/burn transaction.
+    , monetaryPolicyIndex :: !(ApiT DerivationIndex)
+    -- ^ The monetary policy index the asset was minted/burnt under.
+    , policyId            :: !(ApiT W.TokenPolicyId)
+    -- ^ The policy ID the asset was minted/burnt under.
+    , assetName           :: !(ApiT W.TokenName)
+    -- ^ The name of the asset minted/burnt.
+    , subject             :: !(ApiT W.TokenFingerprint)
+    -- ^ The subject of the asset minted/burnt. This is useful to users wishing
+    -- to attach metadata to their asset.
+    }
```

Comment:
How about including the script in the response? Is that feasible?

Reply:
I think so. We could return the JSON representation of the script, or the CBOR, but I think JSON is more appropriate?

Reply:
Addressed in 619ba4806.